install/empty_data.php

```diff
@@ -9479,105 +9492,119 @@ public function getEmptyData(): array
+                    $sub_entities_to_create[] = "E2E worker entity $padded_i";
+
+                    $users_to_create[] = [
+                        'login'       => "e2e_worker_account_$padded_i",
+                        'password'    => password_hash(
+                            "e2e_worker_account_$padded_i",
+                            PASSWORD_DEFAULT,
+                        ),
+                        'realname'    => "E2E worker account $padded_i",
+                        'entities_id' => $next_available_entity_id + $i - 1,
+                    ];
+                }
+            } else {
+                throw new LogicException(); // Can't happen.
+            }
 
-            // Assign e2e user all default profiles on the e2e entity
-            $tables['glpi_profiles_users'][] = [
-                'id' => 6,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_SUPER_ADMIN,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
-            ];
-            $tables['glpi_profiles_users'][] = [
-                'id' => 7,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_SELF_SERVICE,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
-            ];
-            $tables['glpi_profiles_users'][] = [
-                'id' => 8,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_OBSERVER,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
-            ];
-            $tables['glpi_profiles_users'][] = [
-                'id' => 9,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_ADMIN,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
-            ];
-            $tables['glpi_profiles_users'][] = [
-                'id' => 10,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_HOTLINER,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
-            ];
-            $tables['glpi_profiles_users'][] = [
-                'id' => 11,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_TECHNICIAN,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
-            ];
-            $tables['glpi_profiles_users'][] = [
-                'id' => 12,
-                'users_id' => 7,
-                'profiles_id' => self::PROFILE_READ_ONLY,
-                'entities_id' => 1,
-                'is_recursive' => 1,
-                'is_dynamic' => 0,
+            // Create required entites
+            foreach ($sub_entities_to_create as $entity) {
+                $subentity1 = array_replace($root_entity, [
```

Comment:
There is no more multiple `$subentityX` variables.
```suggestion
                $subentity = array_replace($root_entity, [
```